staff/roberto-perez/cv/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Link</title>
+</head>
+<body>
+    <aside>
+        <header>
+            <figure>
+                <img src="" alt="Link">
+            </figure>
+            <h1>Link (リンク Rinku)</h1>
+        </header>
+        <section>
+            <header>
+                <h2>Contacto</h2>
+            </header>
+            <ul>
+                <li>
+                    <address>Hyrule</address>
+                </li>
+                <li>
+                    669 696 696
+                </li>
+                <li>
+                    link@gmail.com
```

Comment:
You can hyperlink mail and phone number to make easier any interaction with your cv.
Try 
```javascript
<a href="tel: ...">... </a>
<a href="mailto: ... @ ... .com"> ... @ ... .com</a>
```

Reply:
Done!